examples/api/src/views/FileSystem.svelte

```diff
@@ -2,16 +2,18 @@
   import * as fs from "@tauri-apps/plugin-fs";
   import { convertFileSrc } from "@tauri-apps/api/core";
   import { arrayBufferToBase64 } from "../lib/utils";
+  import { onDestroy } from "svelte";
 
   export let onMessage;
   export let insecureRenderHtml;
 
   let path = "";
   let img;
+  /** @type {fs.FileHandle} */
   let file;
   let renameTo;
   let watchPath = "";
-  let watchDebounceDelay = 0;
+  let watchDebounceDelay = "0";
```

Comment:
is this right?

Reply:
Yeah it's bind to an input like this, it it's actually a string

```tsx
<input
  class="input"
  id="watch-debounce-delay"
  bind:value={watchDebounceDelay}
/>
```

Reply:
kinda weird ngl, shouldn't it be bound to an input of number type?

Reply:
Yeah probably, but that'll still be a string I think, if svelte didn't do any conversions